### PR TITLE
doc: misc minor fixes

### PR DESCRIPTION
**Pull request by drunkard**

### CODE REVIEW

File: doc/install/install-ceph-gateway.rst [hunk 1]
```diff
@@ -234,12 +234,11 @@ Configure Bucket Sharding
...
 
 A Ceph Object Gateway stores bucket index data in the ``index_pool``, which
-defaults to ``.rgw.buckets.index``. Sometimes users like to put many objects
-(hundreds of thousands to millions of objects) in a single bucket. If you do
-not use the gateway administration interface to set quotas for the maximum
-number of objects per bucket, the bucket index can suffer significant
-performance degradation when users place large numbers of objects into a
-bucket.
+defaults to ``.rgw.buckets.index``. Sometimes, users like to put many objects
+(hundreds of thousands to millions of objects) in a single bucket. If you did
```
Comment (tchaikov) [inline doc/install/install-ceph-gateway.rst]: why s/do/did/?

File: doc/install/install-ceph-gateway.rst [hunk 2]
```diff
@@ -234,12 +234,11 @@ Configure Bucket Sharding
...
+not set quotas for the maximum number of objects per bucket using the gateway
+administration interface, the bucket index can suffer significant
+performance degradation.
```
Comment (tchaikov) [inline doc/install/install-ceph-gateway.rst]: i don't think "when users place large numbers of objects into a bucket" is redundant here. you could put 

> In this case, if you do not ....

to connect the precondition to its consequence.

File: doc/install/install-ceph-gateway.rst [hunk 3]
```diff
@@ -321,7 +321,7 @@ For ``bind``, add a wildcard to the DNS record. For example::
  *       IN      CNAME   @
 
 Restart your DNS server and ping your server with a subdomain to ensure that
-your ``ceph-radosgw`` daemon can process the subdomain requests::
+your DNS configuration works as expected::
```
Comment (tchaikov) [inline doc/install/install-ceph-gateway.rst]: iiuc, the purpose of setting the DNS record is to enable radosgw to process the subdomain request. so the original doc also makes sense.

**Comment by tchaikov** [comment]
the "make check" failure is addressed by #13685

File: doc/install/install-ceph-gateway.rst (hunk 3 above)
Comment (drunkard) [inline doc/install/install-ceph-gateway.rst]: ceph-radosgw is checked in section "Create a Gateway Instance", I think it's clerical mistake here.

File: doc/install/install-ceph-gateway.rst (hunk 1 above)
Comment (drunkard) [inline doc/install/install-ceph-gateway.rst]: It should be "do", you are right.

File: doc/install/install-ceph-gateway.rst [hunk 4]
```diff
@@ -234,11 +234,12 @@ Configure Bucket Sharding
 -------------------------
 
 A Ceph Object Gateway stores bucket index data in the ``index_pool``, which
-defaults to ``.rgw.buckets.index``. Sometimes, users like to put many objects
-(hundreds of thousands to millions of objects) in a single bucket. If you did
-not set quotas for the maximum number of objects per bucket using the gateway
-administration interface, the bucket index can suffer significant
-performance degradation.
+defaults to ``.rgw.buckets.index``. Sometimes users like to put many objects
```
Comment (tchaikov) [inline doc/install/install-ceph-gateway.rst]: could you please just remove the commit being reverted?

**Comment by drunkard** [comment]
Improper commit removed according to advice of @tchaikov

**Pull request merged**